client/apps/edit/components/content2/sections/video/index.jsx

```diff
@@ -59,29 +81,43 @@ export class SectionVideo extends Component {
   }
 
   renderVideoEmbed () {
-    const { section, article, editing } = this.props
-    if (section.get('url') && section.get('url').length) {
+    const { section, article, editing, headerType } = this.props
+    const isBasicHeader = headerType === 'basic-embed'
+    const hasUrl = Boolean(section.get('url'))
```

Comment:
Because 'basic-embed' refers to hiding the preview, it might make sense to use the prop as a boolean rather than a prop specific to the header layout -- i.e.  `<SectionVideo hidePreview />`

Reply:
Ah yes, `hidePreview` is a much better way of doing this 👍